inclusive-design/wecount.inclusivedesign.ca
# feat: add filter to "views" landing page (resolves #174)

## Description

Implement the filter on the "Views" landing page.

## Steps to test

1. Go to "Views";
2. The filter should be displayed on the page under the "search posts" container;
3. Perform filter only, filter then search, search then filter etc.

**Expected behavior:** 

1. The filter action should filter views by tags;
2. The filter with search action should give results that match both the search and filter conditions;
3. Styles should match the figma design.
